.circleci/Dockerfile.integration

```diff
@@ -2,7 +2,7 @@ FROM golang:1.12
 
 # Install docker
 # Adapted from https://github.com/circleci/circleci-images/blob/staging/shared/images/Dockerfile-basic.template
-RUN set -exuo pipefail \
```

Comment:
for some reason the golang image now doesn't support pipefail :roll_eyes: and i didn't spend too long looking into it